packages/discord.js/src/util/Options.js

```diff
@@ -62,6 +62,7 @@ const Transformers = require('./Transformers');
  * @typedef {Object} WebsocketOptions
  * @property {number} [large_threshold=50] Number of members in a guild after which offline users will no longer be
  * sent in the initial guild member list, must be between 50 and 250
+ * @property {number} [version=10] The Discord gateway version to use
```

Comment:
Setting `options.ws.version` is an advanced option and may break the library, since we generally need to refactor several things in order to bump the version. I believe this is the reason why it wasn't documented in the first place.

This PR is fine as-is, but we should add a warning here to let users know.